src/bundle/ui-dev/src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -13,98 +13,101 @@ import {
     loadContentInfo,
     loadLocationsWithPermissions,
 } from './services/universal.discovery.service';
+import {
+    parse as parseTooltips,
+    hideAll as hideAllTooltips,
+} from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/tooltips.helper';
+import { getAdminUiConfig, getTranslator } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
```

Comment:
These helpers are happy, they're together till the end of time! :partying_face: 

Sorry, to much code, my brain explodes...